extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -716,6 +719,10 @@ export class Carousel {
       this.setElementTransform_(spacer, -1, totalLength);
       this.scrollContainer_.appendChild(spacer);
     });
+
+    this.allSpacers_ = this.beforeSpacers_
```

Comment:
nit: `this.beforeSpacers_.concat(this.replacementSpacers_, this.afterSpacers_)`

Reply:
Done